### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MarkHays**

### CODE REVIEW

**Review by tteenz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
 # clippit
 UCB final project
+
+A mock shop called "CLIPPIT" to show our modern booking platform.
+
+Our application is design for barber shop and salons to integrate an easy-to-use scheduling platform with their websites.
```
Comment (jmckenney) [inline README.md]: ~design~ designed

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+
+## Demo Application
+This app is deployed on Heroku please [Click Here](https://clippit-ucb.herokuapp.com/ "Deployed App") to access it
```
Comment (jmckenney) [inline README.md]: Heroku`,` please or Heroku`. ` Please

File: README.md [hunk 3]
```diff
@@ -1,2 +1,52 @@
...
+* Node.js
+* React.js
+* React-Bootstrap
+* Express
+* Mongoose Mongodb
+* Moment.js
+
+
+## ScreenShots
+* Once user enter the website, our home page will show.
```
Comment (jmckenney) [inline README.md]: Once a user enters...
the home page is presented similar to the following screenshot:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,52 @@
...
+<img width="1268" alt="home" src="https://user-images.githubusercontent.com/42901367/54723149-dae6ad00-4b23-11e9-9aeb-95d4c353f140.png">
+
+* In order to book an appointment or to view certain aspect of the site, the user will be prompt to login or sign up if they do not have an account.
```
Comment (jmckenney) [inline README.md]: aspects

File: README.md [hunk 5]
```diff
@@ -1,2 +1,52 @@
...
+<img width="1277" alt="signup" src="https://user-images.githubusercontent.com/42901367/54723673-8cd2a900-4b25-11e9-94de-55cae96a14a9.png">
+
+* Login
+<img width="1269" alt="login" src="https://user-images.githubusercontent.com/42901367/54723690-9fe57900-4b25-11e9-8d98-f0f170944ba5.png">
+
+* Once user have logged in, they can easily go to the book appointment page to schedule their appointment.
```
Comment (jmckenney) [inline README.md]: has

**Review by tteenz — APPROVED** [review]

**Review by tteenz — APPROVED** [review]

**Review by ebresler — APPROVED** [review]

**Comment by ebresler** [comment]
Updated read.me looks good

**Pull request merged**